fabric-network/lib/contract.js

```diff
@@ -197,6 +198,33 @@ class Contract {
 		return listener;
 	}
 
+	/**
+	 * Provide the Discovery Interest settings to help the peer's discovery service
+	 * build an endorsement plan. This interest must include this contract's
+	 * chaincode name and any chaincode names that this chaincode may call.
+	 * The interests must also include the names of collections to which chaincodes
+	 * may belong.
+	 * @param {DiscoveryIntereset[]} interests - These will be used
```

Comment:
Should this be `@param {DiscoveryInterest[]}`?

Reply:
fixed